### DISCUSSION TITLE
Copilot Student Account stay in month plan and can not cancel

### DISCUSSION THREAD

**Discussion by Qiming-Liu** [community/community#46857] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

My account has the `GitHub Student Developer Pack`, so you can see that I can use `GitHub Pro` for free.

A month ago, my `Copilot` suddenly indicated that it was expired and could not be used, so I subscribed to the `Month Plan` and it back to normal. 

Now Copilot is still on bill and cannot cancel the subscription.
![image](https://user-images.githubusercontent.com/68600416/217832868-7c4b626f-d450-434b-a405-a26d0b6d8c48.png)

There is no edit button.

**Comment by iamcjwilliams** [comment, marked as answer]
@Qiming-Liu sorry for the trouble here. Please open a support issue here on our [Support page](https://support.github.com/contact).

**Comment by Qiming-Liu** [comment]
Thanks, the issue has been fixed!
![image](https://user-images.githubusercontent.com/68600416/217959225-4fac4f7b-e06b-4689-9bd0-dec8d4edc70d.png)
